### DISCUSSION TITLE
SkyWalking China Summit 2023 @ Nov. 4th 2023, Shanghai

### DISCUSSION THREAD

**Discussion by wu-sheng** [apache/skywalking#10506] [General]
# Annoucement

SkyWalking China Summit 2023 sponsored by ZMOps and Tetrate @ Shanghai

- Date, Nov 4th, 2023
- Venue, https://j.map.baidu.com/35/vCJ  上海大华虹桥假日酒店
- Registration, https://hdxu.cn/ftjPy

# Historical Events
With the China gov having permanently canceled all COVID-19 strict policies, we are free to run an in-person one-day summit now.

Historically and traditionally, the SkyWalking community only begins an annual DevCon/Summit/Meetup when some vendors propose a sponsorship.DaoCloud

- 2018 sponsored by Microsoft @ Beijing
- 2019 sponsored by China Eastern Airlines & DaoCloud @ Shanghai
- 2020 sponsored by ke.com @ Beijing
- 2021 sponsored by Tencent Cloud @ Beijing
- 2022 sponsored by CSDN as Virtual Summit

At the beginning of 2023, we are opening this sponsorship program. Please submit your proposal here or through mail to `dev@skywalking.apache.org`(public mail list).

The requirements are 
- Summit venue for one day or half of the day for 100-200 attendees.
- Some budgets for speakers. People like me have a company budget to travel if the event happens out of Beijing, but others may not.
- Major cities in China are all acceptable. _We just want to be convenient for the attendees._

**Comment by wu-sheng** [comment]
# [Closed] These two vendors will not be our sponsors.

We have a joint application from http://www.grandage.cn/ [1] and https://www.walkingfunny.com/ [2]

[1] Zabbix service agency
[2] A new startup in the monitoring field has a re-distributed SkyWalking in its product package.

Ref origin, https://lists.apache.org/thread/fdx0b9nl7gxmqnkrkrrcc15tw4mv8f5v

**Reply by wu-sheng** [reply]
An update, **grandage** had given up the joint application. **walkingfunny** mentioned they are trying to host on their own.
I will keep tracking this.

**Reply by wu-sheng** [reply]
walkingfunny.com doesn't have the budget for this sponsorship.

**Comment by wu-sheng** [comment]
walkingfunny just asked for a new round of meetings about their independent sponsorship to build this summit. I will update here if I could work with them to get more details.

**Comment by wu-sheng** [comment]
We have the first discussion. And we plan to host the in-person SkyWalking Summit on Oct. 2023.
It would be a one-day, 8 sessions event. The SkyWalking PMC would be a part of the summit committee to select CFPs and help with the event details. 2 sessions would be sponsor(WalkingFunny Inc.) talks.

**Comment by wu-sheng** [comment]
We finalized the date and location
- Date, Nov 4th, 2023
- Venue, https://j.map.baidu.com/6d/my  上海中庚聚龙酒店

**Reply by cxntsh** [reply]
Yes, as the organizer of this year's SkyWalking conference, I would like to thank Mr. Wu Sheng for his support, and welcome developers from the community and everyone who uses SkyWalking to participate.

**Reply by heyanlong** [reply]
Which registration method is it?

**Reply by wu-sheng** [reply]
We have two registration channels

- CFP for speakers
- Attendees registration

I will work with the sponsor to get these work.

**Reply by zhaoyunxing92** [reply]
how to sign up

**Reply by wu-sheng** [reply]
Be patient. If you are looking for CFP, it comes soon.
The attendee registration will be later.

**Reply by wu-sheng** [reply]
@zhaoyunxing92 We have the registration page now, https://hdxu.cn/ftjPy

**Comment by wu-sheng** [comment]
The event is super successful. We have 100-200 people joined the in person anniversary summit.

Glad to have the party with many committers, contributors, and users.

See you all next year.

![image](https://github.com/apache/skywalking/assets/5441976/f6607221-e401-4e69-8209-648986183686)

About the recorded presentations, we will post them when it is ready.
